bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-name/component.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { defineMessages } from 'react-intl';
+import Icon from '/imports/ui/components/icon/component';
+import styles from './styles';
+
+
+const messages = defineMessages({
+  presenter: {
+    id: 'app.userList.presenter',
+    description: 'Text for identifying presenter user',
+  },
+  you: {
+    id: 'app.userList.you',
+    description: 'Text for identifying your user',
+  },
+  locked: {
+    id: 'app.userList.locked',
+    description: 'Text for identifying locked user',
+  },
+  guest: {
+    id: 'app.userList.guest',
+    description: 'Text for identifying guest user',
+  },
+  menuTitleContext: {
+    id: 'app.userList.menuTitleContext',
+    description: 'adds context to userListItem menu title',
+  },
+  userAriaLabel: {
+    id: 'app.userList.userAriaLabel',
+    description: 'aria label for each user in the userlist',
+  },
+});
+const propTypes = {
+  user: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    isPresenter: PropTypes.bool.isRequired,
+    isVoiceUser: PropTypes.bool.isRequired,
+    isModerator: PropTypes.bool.isRequired,
+    image: PropTypes.string,
+  }).isRequired,
+  compact: PropTypes.bool.isRequired,
+  intl: PropTypes.shape({}).isRequired,
+  meeting: PropTypes.shape({}).isRequired,
+  isMeetingLocked: PropTypes.func.isRequired,
+};
+
+const UserName = (props) => {
+  const {
+    user,
+    intl,
+    compact,
+    isMeetingLocked,
+    meeting,
+  } = props;
+
+  if (compact) {
+    return null;
+  }
+
+  const userNameSub = [];
+
+  if (compact) {
+    return null;
+  }
+
+  const isViewer = !(user.isPresenter || user.isModerator);
+
+  if (isMeetingLocked(meeting.meetingId) && isViewer && user.isLocked) {
```

Comment:
I know that this isn't related to this PR but could you remove this and add to the `user.isLocked` in the service?

imports/ui/services/user/mapUser.js
```
isLocked: user.locked && meeting.isLocked && !(user.isPresenter || user.isModerator),
```